pkg/kv/kvserver/replica_application_result.go

```diff
@@ -560,15 +556,14 @@ func (r *Replica) handleTruncatedStateResult(
 	// reasons.
 	// TODO(#136416): If a crash occurs before the files are durably removed,
 	// there will be dangling files at the next start. Clean them up at startup.
-	r.handleRaftLogDeltaResult(raftLogDelta, isRaftLogTruncationDeltaTrusted)
 }
 
-// TODO(pav-kv): update the stats in the same Replica.mu section with the
-// truncated state update.
+// handleRaftLogDeltaResult updates the raft log stats with the given delta.
+// Both Replica.{raftMu,mu} must be held.
 func (r *Replica) handleRaftLogDeltaResult(delta int64, isDeltaTrusted bool) {
```

Comment:
We usually have a `somethingLocked` suffix on the method in that case, any reason you don't do this here?

Reply:
No reason, just keeping the old names. Added a commit with renames.